portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,259 @@
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Skills section -->
+    <section id="skills" class="p-5 bg-dark text-center text-md-start">
+        <div class="container bg-dark text-light">
+            <div class="row align-items-center justify-content-between">
+                <!-- Info column -->
+                <div class="col-md px-5 pb-sm-0">
+                    <h1 class="pb-3">My Skills :0</h1>
+                    <p class="lead">I'm a <span class="font-weight-bold text-warning">fast learner</span>, I have superb <span class="text-warning">logical reasoning</span> and <span class="text-warning">critical thinking</span> abilities and I possess the mysterious abilty to be a somewhat social introvert.</p>
+                    <p class="lead">But enough about what I think, here's my technical skillset too.</p>
+                </div>
+                <!-- Skill list column -->
+                <div class="col-md p-3">
+                    <ul class="list-group bg-dark">
+                        <li class="list-group-item bg-success text-center text-w text-light h5">C/C++</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Python</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">C#</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">HTML/CSS/JavaScript</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Elixir/Racket</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Unity</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Solidworks</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Random quote section -->
+    <section class="p-5 bg-primary text-center text-md-start text-light" id="randomQuote">
+        <div class="container bg-primary text-light">
+            <div class="row align-items-center justify-content-between">
+                <!-- Text section -->
+                <div class="col-md">
+                    <h1 class="pb-3">Random Quote :P</h1>
+                    <p>If you're bored, hear a random quote from my favourite horror movies.</p>
+                </div>
+                <!-- Quote section -->
+                <div class="col-md lead text-center">
+                    <button type="button" class="btn btn-warning" onclick="addQuote()"><i class="bi bi-chat-square-quote text-light"></i></button>
+                    <p class="pt-2" id="quote"></p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Contact me section -->
+    <section class="p-5 text-center" id="contactMe">
```

Comment:
[3.1.9](https://google.github.io/styleguide/htmlcssguide.html#id_Attributes): Where id attributes are strictly required, always include a hyphen in the value